extensions/sql-database-projects/src/dialogs/publishDatabaseDialog.ts

```diff
@@ -145,6 +145,7 @@ export class PublishDatabaseDialog {
 			this.connectionRow = this.createConnectionRow(view);
 			this.databaseRow = this.createDatabaseRow(view);
 			const displayOptionsButton = this.createOptionsButton(view);
+			displayOptionsButton.enabled = false;
```

Comment:
shouldn't need this if the button isn't getting added to the form container

Reply:
This is added to avoid commenting multiple lines of code, If i don't add, compiling error is coming variable displayOptionsButton  is declared but not used, so i have to comment a method and variable declaration and eventually tests! so just added this!